src/mscorlib/src/System/TypedReference.cs

```diff
@@ -85,13 +85,13 @@ public override bool Equals(Object o)
             throw new NotSupportedException(SR.NotSupported_NYI);
         }
 
-        public unsafe static Object ToObject(TypedReference value)
+        public static unsafe Object ToObject(TypedReference value)
         {
             return InternalToObject(&value);
         }
 
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        internal unsafe extern static Object InternalToObject(void* value);
+        internal static extern unsafe Object InternalToObject(void* value);
```

Comment:
We don't seem to specify ordering beyond visibility first. Seems like we're settling on something.

It reminds me of https://en.wikipedia.org/wiki/Adjective#Order

Reply:
The coding guidelines specify order for readonly. For the unspecified ones, I have unified them on the order that majority of the code in corefx has been using.